### DISCUSSION TITLE
Agent Based Installer stuck on OCI

### DISCUSSION THREAD

**Discussion by fcolomas** [okd-project/okd#1967] [General]
Are you guys having troubles with the agent based installer? I got a Master node which stops to provision at this lines:

Jul 09 05:11:00 ddmstr01.sndamdocsv360.vcndesarrollo.oraclevcn.com next-step-runner[2037]: WARN[0000] Agent Authentication Token not set           
Jul 09 05:11:00 ddmstr01.sndamdocsv360.vcndesarrollo.oraclevcn.com podman[2026]: 2024-07-09 05:11:00.371422431 +0000 UTC m=+0.017063510 image pull 618fac0c02a3c053fdc4ec9905da9417afe996cc8db3dc0a8875c3b309d3ecb0 quay.io/openshift/okd-content@sha256:57109646c2e66aee05c7003d0e0b7f1538f37a01c2f633fad8e962b3e1727335

And it never move from it, if you try to pull that image it says 404, any clues?
Latest OKD 4.15 installer used to create the image

Install

apiVersion: v1
baseDomain: validdomain.com
metadata:
  name: sndamdocsv360
networking:
  clusterNetwork:
  - cidr: 10.128.0.0/14
    hostPrefix: 23
  networkType: OVNKubernetes
  machineNetwork:
  - cidr: 10.201.16.0/24
  serviceNetwork:
  - 172.30.0.0/16
compute:
- architecture: amd64
  hyperthreading: Enabled
  name: worker
  replicas: 0
controlPlane:
  architecture: amd64
  hyperthreading: Enabled
  name: master
  replicas: 3
platform:
   external:
    platformName: oci
    cloudControllerManager: External
sshKey: ssh-ed25519 AVALID SSHKEY validuser@validdomain.com
pullSecret: 'A valid pull secret from RH' 

agent

apiVersion: v1alpha1
metadata:
  name: sndamdocsv360
rendezvousIP: 10.201.16.67

**Comment by ArthurVardevanyan** [comment]
OKD 4.15 SCOS Stable with Secure Boot Disabled Seems to work.
https://github.com/ArthurVardevanyan/HomeLab/tree/main/kubevirt/okd

**Comment by fcolomas** [comment]
Can you please point the url to get that installer to test?
________________________________

**Reply by ArthurVardevanyan** [reply]
> Can you please point the url to get that installer to test?


https://amd64.origin.releases.ci.openshift.org/#4-scos-stable

**Comment by JaimeMagiera** [comment]
Adding this here and in the slack thread for completeness...

Secure Boot is a known issue. We need to add it to our FAQ. This week, I'm hoping to get clarity on where things stand.
https://issues.redhat.com/browse/RHEL-4391
